### ISSUE TITLE
1.9 release

### ISSUE THREAD

**Issue by guillaumebriday** [awesome-print/awesome_print#399]
According to the changelog : https://github.com/awesome-print/awesome_print/blob/master/CHANGELOG.md#190

The 1.9 has been released but it's not available on rubygems.

Could you release it please?

Thanks 👍

**Comment by BryanH** [comment]
Yes! I missed a setting required for rubygems, so it will be 1.9.1. Should have it pushed this week!

**Comment by guillaumebriday** [comment]
Great thanks @BryanH

**Comment by tbsvttr** [comment]
@BryanH Thanks. :)

**Comment by BryanH** [comment]
Ugh, with TravisCI.org shuttting down, the merge-test-release cycle is VERY slow. Testing is taking 15-20 hours for _each merge_.

I just published 1.9.1 and will request a push to rubygems as soon as it passes TravisCI testing

**Comment by guillaumebriday** [comment]
Awesome print should probably move to Github Actions, what do you think?

1.9.0 is not available yet on https://rubygems.org/gems/awesome_print/versions

**Comment by BryanH** [comment]
@guillaumebriday 100% agree, now that TravisCI is [shutting down/severely limiting the FOSS tier](https://travis-ci.community/t/org-com-migration-unexpectedly-comes-with-a-plan-change-for-oss-what-exactly-is-the-new-deal/10567).

**Comment by BryanH** [comment]
Good news everyone, we're on track to release 1.9.x to Rubygems. I totally understand the frustration--and share it--and appreciate your hanging in there.

**Comment by BryanH** [comment]
One of the tests is still failing on Ruby 2.7.x - once that's resolved we'll be good to go

**Comment by guillaumebriday** [comment]
@BryanH If we can help feel free to ask! 👍

**Comment by keyboard-clacker** [comment]
@BryanH No pressure here, just curious if there's any way I could help as well; happy to lend a hand if you all need it! Looking forward to the 1.9 release

**Comment by BryanH** [comment]
@keyboard-clacker @guillaumebriday The ruby-head tests are failing; I marked them ignore for now, but if someone could submit a PR  so ruby-head passes, that would be really helpful

**Comment by n-rodriguez** [comment]
For what it worth, there is a maintained fork here : https://github.com/amazing-print/amazing_print
